src/lightning_app/cli/cmd_apps.py

```diff
@@ -63,6 +63,14 @@ def list(self, cluster_id: str = None, limit: int = 100) -> None:
         console = Console()
         console.print(_AppList(self.list_apps(cluster_id=cluster_id, limit=limit)).as_table())
 
+    def delete(self, app_id: str) -> None:
```

Comment:
please add support for _multiple_ app ids so we can batch delete from the CLI as well.